docs/development/codeguide.rst

```diff
@@ -758,15 +560,15 @@ Catching of exceptions should always use this syntax::
     try:
```

Comment:
You can probably delete the whole section because the old syntax `except exception, e:` isn't allowed anymore.

Reply:
Good catch. It is removed. Thanks!